packages/web-app-files/src/components/SideBar/Shares/Collaborators/ListItem.vue

```diff
@@ -64,7 +64,11 @@
               />
             </div>
             <div v-else-if="share.role">
-              <span class="oc-mr-xs" v-text="share.role.label" />
+              <span
+                v-oc-tooltip="share.role.description(false)"
```

Comment:
Ah lol, now I see why we have untranslated strings in this area. The role doesn't do translation internally. You need to wrap the description and the label (2 lines below) into a `$gettext` call, i.e. `$gettext(share.role.description(false))` and `$gettext(share.role.label)`